### ISSUE TITLE
Semantic-Release 15.13.18 wrongly states that an old version of GIT is installed

### ISSUE THREAD

**Issue by mjenny** [semantic-release/semantic-release#1226]
## Current behavior
Since updating to semantic-release 15.13.18 we are receiving the message that we do have an outdated GIT version (below 2.0.0) although we are using GIT 2.19.0.
This leads to the command `npx semantic-release` existing with an error.

This worked with semantic-release version 15.13.17.

## Expected behavior
semantic-release shouldn't fail since we are using GIT 2.19.0.

## Environment

- **semantic-release** version: 15.13.18
- CI environment: Jenkins on OpenShift
- Plugins used: ["@semantic-release/commit-analyzer", "@semantic-release/release-notes-generator", "@semantic-release/changelog", "@semantic-release/npm", "@semantic-release/git"]
- **semantic-release** configuration: 
```{
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    ["@semantic-release/changelog", {
      "changelogFile": "docs/CHANGELOG.md"
    }],
    ["@semantic-release/npm", {
      "npmPublish": false,
      "tarballDir": "dist"
    }],
    ["@semantic-release/git", {
      "assets": ["dist/**/*.{js,css}", "docs", "package.json"],
      "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
    }]
  ],
  "branch": "master",
  "dryRun": false
}
```

- CI logs:
```+ git --version
git version 2.19.0
[...]
+ npx semantic-release
 [semantic-release]: Git version 2.0.0 is required. Found 1.8.3.
 [2:47:55 PM] [semantic-release] ��� ���  Running semantic-release version 15.13.18
```

**Comment by pvdlg** [comment]
We run `git --version` to figure out the version installed.
So it seems in the environment in which you are running semantic-release, you are using git version `1.8.3`.

Your CI says `git version 2.19.0` but I'm guessing the `git` binary in your `PATH` is not that one and is an older version.

**Comment by mjenny** [comment]
@pvdlg 
Thank you for your response. I'm just curious if you changed anything particular between 15.13.17 and 15.13.18 since we don't have an issue on 15.13.17.

**Comment by mjenny** [comment]
@pvdlg 
Nevermind, I fixed it by forcefully removing `git 1.8.3` which was installed somehow.
